ibmdbpy/learn/association_rules.py

```diff
@@ -49,25 +49,79 @@ def __init__(self, modelname = None, minsupport = None, maxlen = 5, maxheadlen =
             The name of the Association Rules model that is built. If the 
             parameter corresponds to an existing model in the database, it 
             will be replaced during the fitting step.
+        
         minsupport : float or integer, optional
             The minimum fraction (0.0 - 1.0) or the minimum number (above 1) of
             transactions that must contain a pattern to be considered as frequent.
             Default: system-determined
             Range: >0.0 and <1.0 for a minimum fraction
                    >1 for a minimum number of transactions.
+        
         maxlen : int, optional, >=2, default: 5
             The maximum length of a pattern or a rule, that is,
             the maximum number of items per pattern or rule.
+        
         maxheadlen : int, optional, >= 1 and <maxlen, default: 1
             The maximum length of a rule head, that is, the maximum number of
             items that might belong to the item set on the right side of a rule.
             Increasing this value might significantly increase the number of detected rules.
+        
         minconf : float, optional, >=0.0 and <= 1, default: 0.5
             The minimum confidence that a rule must achieve to be kept in the model of the pattern.
             
         Attributes
         ----------
-        TODO
+        
+        _idadf : Gets set at fit step. Input table, IdaDataFrame the model will be fit on;
```

Comment:
Please don't document attributes beginning with an underscore. These are not public (see https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables).

Reply:
yes right